prbt_default_application/launch/my_application.launch

```diff
@@ -59,9 +59,15 @@ limitations under the License.
 
       <!-- Safe-torque-off-input from the safety controller is read to brake the robot before power-off
            The signal is read via modbus and you can configure the modbus server ip -->
-      <!-- <arg name="sto" value="pnoz" /> -->
+      <!-- <arg name="sto" value="pss4000" /> -->
       <!-- <arg name="sto_modbus_server_ip" value="192.168.0.10" /> -->
 
+      <!-- You can additionally select the two safety functions below: -->
+      <!-- Set braketest usage -->
```

Comment:
The comment should explain
* When would you want to enable this? Real-hardware only?
* What effect does this have and relation between the features e.g. "read via modbus, so you need to also set the sto_modbus_server_ip above" --> consistent naming??